apps/component-generator/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "component-generator",
```

Comment:
I wonder if this package should be private as well? I think we could keep it internal to the FURN repo. What do you think?

Reply:
Yes, I agree. Added private prop